Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx

```diff
@@ -168,6 +168,22 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
     }
   }
 
+  private renderSendingText = (message: any) => {
+    if (message.isPreparingToSend) {
+      return (
+        <Text
+          truncated
+          content="preparing to send"
```

Comment:
Can you please use "Preparing to send..." in stead? so that I will align with the "Sending..." message.

Reply:
updated